ignite/contrib/handlers/param_scheduler.py

```diff
@@ -468,9 +482,8 @@ def __init__(self, schedulers, durations, save_history=False):
         self.schedulers = schedulers
         self.durations = durations
 
-        self.optimizer = self.schedulers[0].optimizer
-        if not (all(id(s.optimizer) == id(self.optimizer) for s in self.schedulers)):
```

Comment:
I think the check here is valid as we unlikely want to concat lr schedulers working on different optimizers such that at first the first scheduler is applied on the first optimizer and then the second one on another optimizer.

Reply:
let me think about it

Reply:
Sounds good